Source/CloudyWebAPI/Private/CloudyWebAPI.cpp

```diff
@@ -302,21 +303,15 @@ void CloudyWebAPIImpl::ReadAndStoreSaveFileURL(FString JsonString, int32 PlayerC
     TSharedRef<TJsonReader<TCHAR>> JsonReader = TJsonReaderFactory<TCHAR>::Create(JsonString);
     FJsonSerializer::Deserialize(JsonReader, JsonObject);
 
-    if (PlayerControllerId == 0)
-    {
-        SaveFileURL0 = JsonObject->GetStringField("saved_file");
-    }
-    else if (PlayerControllerId == 1)
+    // More player controllers than the TArray size
+    if (PlayerControllerId >= SaveFileUrls.Num())
```

Comment:
This line is comparing the same as line 259. If true, it would already have returned on line 262.